polymer.json

```diff
@@ -25,7 +25,7 @@
   ],
   "lint": {
     "rules": ["polymer-3"],
-    "ignoreWarnings": ["could-not-resolve-reference"],
+    "ignoreWarnings": ["could-not-resolve-reference", "cant-resolve-module-specifier"],
```

Comment:
Why are you ignoring this one? I much rather have you ignore all the React stuff from Polymer Lint.

Reply:
How can I do that ? (Sorry, I'm quite bad with javascript :( )